server/issue.go

```diff
@@ -449,6 +449,14 @@ func (s *Server) createIssue(ctx context.Context, issueCreate *api.IssueCreate,
 		return issue, nil
 	}
 
+	if _, err := s.ScheduleNextTaskIfNeeded(ctx, issue.Pipeline); err != nil {
+		return nil, fmt.Errorf("failed to schedule task after creating the issue: %v. Error %w", issue.Name, err)
+	}
+	// We need to re-compose the issue relationship since schedule next task to run may change the issue internal fields.
+	if err := s.composeIssueRelationship(ctx, issue); err != nil {
```

Comment:
I still don't quite understand the full story. The creator, updater should always exist. ScheduleNextTaskIfNeeded should not nullify it unless it's a bug.

Reply:
Done.